File: java/kata/src/test/java/br/com/ari/algo/algorithms/TwoSumTest.java
package br.com.ari.algo.algorithms;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class TwoSumTest {

    @Test
    void testTwoSum() {
        int[] nums = new int[]{ 2,7,11,15 };
        int target = 9;

        int[] result = TwoSum.solve(nums, target);

        assertThat(result).isEqualTo(new int[]{ 0, 1 });
    }

}

File: go/kata/algorithms/two_sum_test.go
package algorithms

import (
	"github.com/stretchr/testify/assert"
	"testing"
)

func TestTwoSum(t *testing.T) {
	nums := []int{2, 7, 11, 15}
	target := 9
	result := TwoSum(nums, target)

	assert.Equal(t, [2]int{0, 1}, result)
}
